test/test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# # License and Copyright
+#
+# Copyright: Copyright (c) 2016 Chef Software, Inc.
+# License: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# The list of all specs to run after basic and env tests are run.
+# WITHOUT .rb suffix.
+all_specs=(crypto)
+
+# sadly, this is NOT a banner of a cat. But, with a pull request,
+# YOU could make it so.
+cat banner
+
+# load in common test env vars
+HAB=/bin/hab
```

Comment:
Should we try to look for this by using their `PATH` instead? Our OSX installer puts the binary to `/usr/local/bin/hab` instead of `/bin/hab`.

(actually, should I be running this test suite on my local machine or inside a docker container?)